pkg/detectors/azurecontainerregistry/azurecontainerregistry.go

```diff
@@ -94,6 +95,10 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 	return results, nil
 }
 
+func (s Scanner) IsFalsePositive(_ detectors.Result) bool {
+	return false
+}
```

Comment:
Did you consider creating a struct that implements this so it can be embedded into the `Scanner` struct?

```go
// in detectors package
type SkipFalsePositiveCheck struct{}

func (SkipFalsePositiveCheck) IsFalsePositive(Result) bool {
    return false
}

// elsewhere

type Scanner struct {
     detectors.SkipFalsePositiveCheck
}
```

Reply:
no, that's a much better idea!

Reply:
Marginally? imo it's a "nice-to-have" for a common scenario, especially for areas of the code that get a lot of outside contribution.